pkgs/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_function_declaration.dart

```diff
@@ -91,6 +91,22 @@ ParsedFunctionInfo parseFunctionInfo(
 
   final openParen = tokens.indexWhere((tok) => matchFragment(tok, 'text', '('));
   if (openParen == -1) throw malformedInitializerException;
+
+  final prefixAnnotations = <String>{};
+  while (true) {
+    final tok = tokens.first['kind'].get<String>();
```

Comment:
Rather than manually checking the token kind, then calling `maybeConsume('keyword')`, you can simply call `maybeConsume('keyword')` here. If the kind is not `'keyword'`, it will return null, and won't consume the token. So you can call `final keyword = maybeConsume('keyword');` here and change `if (tok == 'keyword') {` to `if (keyword != null) {`.